### PR TITLE
bpo-25597: Ensure wraps' return value is used for magic methods in MagicMock

### PR DESCRIPTION
**Pull request by tirkarthi**
https://bugs.python.org/issue25597

### CODE REVIEW

**Review by mangrisano — APPROVED** [review]
LGTM.

File: Misc/NEWS.d/next/Library/2019-09-12-12-11-05.bpo-25597.mPMzVx.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Ensure ``wraps`` is supplied to :class:`unittest.mock.MagicMock` is used to
+calculate return values for the magic methods instead of using the default
```
Comment (cjw296) [inline Misc/NEWS.d/next/Library/2019-09-12-12-11-05.bpo-25597.mPMzVx.rst]: "Ensure, if ``wraps`` is supplied to :class:`unittest.mock.MagicMock`, it is used to" maybe?

**Review by cjw296 — CHANGES_REQUESTED** [review]
Great, but the news entry needs some work, not sure my interpretation is correct?

File: Misc/NEWS.d/next/Library/2019-09-12-12-11-05.bpo-25597.mPMzVx.rst (hunk 1 above)
Comment (tirkarthi) [reply Misc/NEWS.d/next/Library/2019-09-12-12-11-05.bpo-25597.mPMzVx.rst]: Thanks Chris, updated.

**Comment by tirkarthi** [comment]
I have made the requested changes; please review again

**Review by cjw296 — APPROVED** [review]

**Comment by tirkarthi** [comment]
Thanks Chris :)

**Pull request merged**